openrr-client/src/clients/collision_check_client.rs

```diff
@@ -131,45 +42,20 @@ where
         positions: Vec<f64>,
         duration: std::time::Duration,
     ) -> Result<WaitFuture, Error> {
-        self.collision_checker.check_joint_positions(
-            &self.current_joint_positions()?,
-            &positions,
-            duration,
-        )?;
+        self.collision_checker
+            .check_joint_positions(&self.current_joint_positions()?, &positions, duration)
+            .map_err(|e| Error::Other(e.into()))?;
         self.client.send_joint_positions(positions, duration)
     }
 
     fn send_joint_trajectory(&self, trajectory: Vec<TrajectoryPoint>) -> Result<WaitFuture, Error> {
-        self.collision_checker.check_joint_trajectory(&trajectory)?;
+        self.collision_checker
+            .check_joint_trajectory(&trajectory)
+            .map_err(|e| Error::Other(e.into()))?;
```

Comment:
It's better not to return `Other`. Please return some information.

Reply:
To do it, we have to change `arci::Error`; on the other hand, `arci` should not depend on `openrr*`.